src/deepak/codingExam5/Program2.java

```diff
@@ -0,0 +1,34 @@
+package deepak.codingExam5;
+
+public class Program2 implements Manager {
+	
+	//@Override
+	public int[] findTargetIndex(int[] arr, int target) {
+		int[] output = new int[2];
+		for (int i=0; i<arr.length-1; i++) {
```

Comment:
for(){
   for(){
       if(target==I=j){
             //code
       }
  }
}